pillarbox-core-business/build.gradle.kts

```diff
@@ -2,6 +2,7 @@
  * Copyright (c) 2022. SRG SSR. All rights reserved.
  * License information is available from the LICENSE file.
  */
+@Suppress("DSL_SCOPE_VIOLATION") // TODO: Remove once KTIJ-19369 is fixed
```

Comment:
⚠️ **[detekt]** <detekt.ForbiddenComment> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>This comment contains 'TODO:' that has been defined as forbidden in detekt.